### DISCUSSION TITLE
decode encrypted link

### DISCUSSION THREAD

**Discussion by Alantienviet** [iptv-org/community#214] [Q&A] [answered]
can any master decipher this encrypted link...it has a connection time limit of 6 hours

https://106bec9d8.vws.vegacdn.vn/0d93205c38ab4946c24593906c49580e66980246/1632988206/cliptv_live/amlst:planetearthhd/chunklist_b3128000.m3u8

**Comment by BellezaEmporium** [comment]
1) From which website did you get it from 
2) IP-locked ?
3) Encoded data sounds like SHA-1 to me.

**Comment by Sphinxroot** [comment]
could be
https://www.vtvcab.vn/channel/planet-earth-hd-1,PLANET_EARTH_HD.html
or here
https://cliptv.vn/


both geo

**Comment by Alantienviet** [comment]
I got it from vtvcab.vn

Vào 01:18, T.3, 5 Th10, 2021 Sphinxroot ***@***.***> đã viết:

**Comment by BellezaEmporium** [comment, marked as answer]
Sounds like a succession of tokens and links to get to the final link.

You simply wanted to know what's inside the sha-1 ? Nearly impossible until you know the combination behind.

Or is it something you would want added inside our playlists ?

**Reply by Alantienviet** [reply]
i want to add it to playlist...but it seems the link is encrypted and i don't know how to undo that 6 hour link time limit
